templates/plan_template.html

```diff
@@ -149,6 +149,10 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
             <tbody id="results-table-body">
             </tbody>
         </table>
+
+        <div class="d-flex d-none" id="no-valid-plan-error-msg">
```

Comment:
We may also consider using `alert` class in Bootstrap. It could help give consistent alert msg styles across different places, in case we add new ones in the future. 
https://getbootstrap.com/docs/5.0/components/alerts/

Reply:
Updated to use the alerts, looks so much better!